app/assets/javascripts/boot.coffee

```diff
@@ -11,6 +11,19 @@ drawMenuPlanRecipeTotal = ->
   totalRecipes = window.currentMenuPlan.get('recipeIds').length
   $('.recipe-count').text(totalRecipes)
 
+drawMenuPlanDropdown = ->
+  navDropdown = document.getElementById("menu_plan_dropdown")
+  ReactDOM.render React.createElement(MenuPlanDropdown, recipes: window.currentMenuPlanRecipes, deleteRecipe: removeRecipe), navDropdown
+
+removeRecipe = (recipe) ->
+  id = recipe.get('id')
+  window.currentMenuPlan.removeRecipeId(id)
+  window.currentMenuPlanRecipes.remove({id: id})
```

Comment:
Sounds good 👍
I'd kept them both around for awhile just to avoid breaking existing functionality and I hadn't seen a clear way forward yet.

I like this solution though!